PYME/experimental/_triangle_mesh.pyx

```diff
@@ -2075,6 +2077,7 @@ cdef class TriangleMesh(TrianglesBase):
             # TODO - move this to a helper function
             self._vertices['locally_manifold'] = 1
             self._vertices['locally_manifold'][self._halfedges['vertex'][self._halfedges['twin'] == -1]] = 0
+            self._vertices['locally_manifold'][-1] = (self._halfedges['vertex'][-1] != -1) and (self._halfedges['twin'][-1] == -1)
```

Comment:
???? why do we need a special case here?

Reply:
I think I wrote this in case you ran into a case where `self._halfedges['vertex'][self._halfedges['twin'] == -1]` returned a -1 (not impossible). In this case, `self._vertices['locally_manifold'][-1] = 0`. However, `self._vertices['locally_manifold'][-1]` might in fact be be manifold. So, you'd treat it as non-manifold unnecessarily. It's a silly little check (and I think I meant to write `self._halfedges['twin'][-1] != -1`). I'm not sure most of these meshing tweaks are relevant anymore.